src/test/java/com/scottlogic/SortingTests/AuthorSortTest.java

```diff
@@ -14,7 +14,7 @@
 import java.util.List;
 
 
-public class AuthorSortingTests {
+public class AuthorSortTest {
```

Comment:
Class is called SortAuthor, test should be SortAuthorTest.
They should also have the same package path (com.scottlogic....)

Reply:
Fixed